### ISSUE TITLE
bitwise or (pipe symbol) in property declaration breaks file documentation

### ISSUE THREAD

**Issue by c-holtermann** [phpDocumentor/phpDocumentor2#1829]
When running this file through phpdoc 2.8.5 

``` php
<?php
/**
 * description file
 */

/**
 * description class c
 */

class c {
 /**
 * description property a
 */
 public $a = 1 | 2;
}

?>
```

the result is this error:
Parse Error: Syntax error, unexpected '|', expecting ',' or ';' on line 14  No summary was found for this file

Removing the pipe symbol the documentation process works without a problem:
``` php
<?php
/**
 * description file
 */

/**
 * description class c
 */

class c {
 /**
 * description property a
 */
 public $a = 1;
}

?>
```

This seems to be a bug. 

1. The error message "No summary was found for this file" is wrong
1. a bitwise or at this property declaration works fine in php.

**Comment by jaapio** [comment]
please try version [2.9.0](https://github.com/phpDocumentor/phpDocumentor2/releases/tag/v2.9.0) and let me know if this problem still exists.

**Comment by c-holtermann** [comment]
The issue has been resolved in Version 2.9.0. Thanks.
